docs/index.rst

```diff
@@ -19,9 +19,42 @@ Features:
 * Minimizes the size of the NWB files by automatically applying chunking and lossless compression.
 * Supports ensembles of multiple data streams, and supports common methods for temporal alignment of streams.
 
-.. seealso::
 
-  Watch a video introduction to NeuroConv `here <https://youtu.be/QofVU-59Dd4>`_
+
+How to use the documentation
+----------------------------
+
+Our documentation is structured to cater to users ranging from beginners to advanced developers and contributors.
+Below is an overview of the key sections to help you navigate our documentation effectively
+
+* **Getting Started: Conversion Examples Gallery**
+
+  If you're new to NeuroConv or NWB, start with the `Conversion Examples Gallery <conversion_examples_gallery/conversion_example_gallery>`_.
+  This section provides concise scripts for converting data from common formats (e.g., Blackrock, Plexon, Neuralynx) to NWB. It's designed to get you up and running quickly.
+
+* **User Guide**
+
+  The `User Guide <user_guide/user_guide>`_ offers a comprehensive overview of NeuroConv's data model and functionalities.
+  It is recommended for users who wish to understand the underlying concepts and extend their scripts beyond basic conversions.
+
+* **Catalogue of Projects**
+
+  `The Catalogue of Neuroconv Projects <catalogue/catalogue>`_ section showcases a collection of successful conversion projects utilizing NeuroConv.
+  It serves as both inspiration and a practical reference for what can be achieved with our library.
+
+* **Developer Guide**
+
+  For developers interested in contributing to NeuroConv, the `Developer Guide <developer_guide>`_ provides essential information such as
+  instructions for building your own classes,  our coding style, instructions on how to build the documentation,
+  run the testing suite, etc.
+
+* **API Reference**
+
+  Detailed documentation of the NeuroConv API can be found in the `API <api>`_ section.
+
+
+Do you find that some information is missing or some section lacking or unclear? Reach out with an issue or pull request on our `GitHub repository <https://github.com/catalystneuro/neuroconv>`_.
+We are happy to help and appreciate your feedback.
 
 .. toctree::
   :maxdepth: 2
```

Comment:
I was suggesting we delete this section bc it's redundant

Reply:
Thanks.
Let's move this to another PR:
https://github.com/catalystneuro/neuroconv/pull/796